internal/pkg/test/resources.go

```diff
@@ -28,8 +29,9 @@ const (
 	TokenTypeKey  = "type"
 	JoinTokenKey  = "joinToken"
 
-	TokenTypeKube = "kube"
-	TokenTypeNode = "node"
+	TokenTypeKube    = "kube"
+	TokenTypeNode    = "node"
+	TokenTypeKubeApp = "kubeapp"
```

Comment:
I think `TokenTypeKubeApp` is not used anymore. This could be replaced with `TokenTypeApp` and used in slices, if you wanted to do that.

Reply:
you are right, I going to replace it now